lib/omniauth/strategies/saml.rb

```diff
@@ -100,25 +95,41 @@ def response_fingerprint
         Digest::SHA1.hexdigest(cert.to_der).upcase.scan(/../).join(':')
       end
 
-      def on_metadata_path?
```

Comment:
Removing `on_metadata_path?` is a breaking change for anyone who has customized that method in a subclass.